reference/latest/src/main/java/com/example/docs/FabricDocsReference.java

```diff
@@ -40,5 +42,8 @@ public void onInitialize() {
 		Registry.register(Registries.PARTICLE_TYPE, Identifier.of(MOD_ID, "sparkle_particle"), SPARKLE_PARTICLE);
 		//#particle_register_main
 		//#entrypoint
+
+		// required to ensure components are registered
+		LOGGER.info(ModComponents.class.getName());
```

Comment:
I recommend bringing the `initialize` method back. Classloading tricks can be confusing even to experienced developers. Look at how [CustomSounds](https://github.com/FabricMC/fabric-docs/blob/main/reference/latest/src/main/java/com/example/docs/sound/CustomSounds.java#L24) handles this same situation if you need inspiration

Reply:
Sure. I had to remove the empty initialize method to make the checkstyle happy so I'll see if there's an alternative